ios/Classes/SwiftZendeskMessagingPlugin.swift

```diff
@@ -23,92 +23,114 @@ public class SwiftZendeskMessagingPlugin: NSObject, FlutterPlugin {
             self.processMethodCall(call, result: result)
         }
     }
-
+    
     private func processMethodCall(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
         let method = call.method
         let arguments = call.arguments as? Dictionary<String, Any>
-        let zendeskMessaging = ZendeskMessaging(flutterPlugin: self, channel: channel)
+        let zendeskMessaging = ZendeskMessaging(flutterPlugin: self)
```

Comment:
I think u don't want to create ZendeskMessaging every time received a method channel call.

Reply:
Done